web_src/js/features/package-editor.js

```diff
@@ -0,0 +1,31 @@
+import $ from 'jquery';
```

Comment:
Linting rules should prevent the addition of new jQuery code, could you utilize vanilla JS instead?

Cc: @silverwind who has more insight into linting

Reply:
There is no linting for it, I don't think it's possible for the linter to distinguish old and new code. But yes, new code should prefer not to use jQuery, we want to eventually remove it.